src/Calendar/components/TimeGrid/TimeGrid.js

```diff
@@ -71,7 +71,7 @@ const TimeGrid = React.forwardRef(
         });
         wrapperRef.current.scrollTop = topOffset;
       }
-    }, [selectedDate, stepMinutes, stepHeight, scrollToTime, wrapperRef]);
+    }, [selectedDate, stepMinutes, stepHeight, wrapperRef]);
```

Comment:
scrollToTime as a dependency here definitely was a regression that caused the extra scrolling